### PR TITLE
impr(UTAPI-96): Switch to scality/actions/action-ssh-to-runner

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -4,8 +4,25 @@ name: tests
...
+        description: Debug (enable the ability to SSH to runners)
+        type: boolean
+        required: false
+        default: 'false'
+      connection-timeout-m:
+        type: number
+        required: false
+        description: Timeout for ssh connection to worker (minutes)
+        default: 30
+      completion-delay-m:
```
Comment (jonathan-gramain) [inline .github/workflows/tests.yaml]: I don't see `completion-delay-m` used anywhere, do you want to keep it?

**Review by jonathan-gramain — APPROVED** [review]

**Review by benzekrimaha — APPROVED** [review]

**Pull request merged**